modules/router/templates/draft-assets.conf.erb

```diff
@@ -32,6 +32,11 @@ server {
   add_header "Access-Control-Allow-Methods" "GET, OPTIONS";
   add_header "Access-Control-Allow-Headers" "origin, authorization";
 
+  # Avoid falling through to the default host if none of our locations match
+  location / {
+    return 404;
+  }
+
```

Comment:
Very minor: I might be tempted to put this location block _below_ the other one, since it's less important. However, I haven't looked at how this is done in other Nginx config files.

Reply:
I don't feel particularly strongly about it so I've left it as it is.